src/app/onboarding/components/create-new-archive/create-new-archive.component.html

```diff
@@ -1,7 +1,91 @@
+        </div>
+      </label>
+
+      <label>
+        <div [ngClass]="getArchiveTypeClasses('type.archive.organization')">
+          <img src="assets/svg/onboarding/archive-organization.svg" alt="Organization Archive" />
+          <div>
+            <strong class="archive-type-name">Organization Archive</strong>
+            <span class="archive-type-description">an organizational entity, such as a company or nonprofit</span>
+          </div>
+          <input type="radio" name="archive-type" value="type.archive.organization" (change)="setArchiveType($event.target.value)" />
+        </div>
+      </label>
+    </div>
+
+    <div class="two-buttons">
+      <button class="btn btn-secondary back-button" (click)="onBackPress()">Back</button>
+      <button class="btn btn-primary next-button" [disabled]="!archiveType" (click)="setScreen('name')">Next: Name Archive</button>
+    </div>
+  </div>
+  <div *ngIf="screen === 'name'">
+    <h2>Create Your First Archive: Name Your Archive</h2>
+    <div class="selected-archive">
+      <ng-container [ngSwitch]="archiveType">
+        <ng-container *ngSwitchCase="'type.archive.person'">
+          <img src="assets/svg/onboarding/archive-person.svg" alt="Person Archive" />
+          <div>
+            <strong class="archive-type-name">Person Archive</strong>
+            <span class="archive-type-description">represents an individual person, such as yourself or someone else</span>
+          </div>
+        </ng-container>
+        <ng-container *ngSwitchCase="'type.archive.family'">
+          <img src="assets/svg/onboarding/archive-group.svg" alt="Group Archive" />
+          <div>
+            <strong class="archive-type-name">Group Archive</strong>
+            <span class="archive-type-description">any group of people, such as a community group or a family</span>
+          </div>
+        </ng-container>
+        <ng-container *ngSwitchCase="'type.archive.organization'">
+          <img src="assets/svg/onboarding/archive-organization.svg" alt="Organization Archive" />
+          <div>
+            <strong class="archive-type-name">Organization Archive</strong>
+            <span class="archive-type-description">an organizational entity, such as a company or nonprofit</span>
+          </div>
+        </ng-container>
+      </ng-container>
+    </div>
+    <p><strong>What is your new archive called?</strong> This is generally the legal or official name of the person, family, or organization the archive is about &mdash; for example, “The Katherine Johnson Archive.”</p>
+    <p>You can always edit your archive’s name later.</p>
+    <div class="name-field">
+      <label for="archive-name">The</label>
+      <input type="text" id="archive-name" class="form-control" (input)="archiveName = $event?.target?.value" [value]="archiveName" [placeholder]="getArchiveNamePlaceholder()" />
```

Comment:
Can we use the [HTML `required` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required)? That'd allow us to lean on the browser for validation, and I believe that instead of needing the input event we could read the value during `onSubmit`.